### PR TITLE
Homework four

### PR DESCRIPTION
**Pull request by TimurIsroilov123**
Added recyclerView for movieList and actors

### CODE REVIEW

File: app/src/main/java/com/bignerdranch/android/androidacademy/FragmentMoviesList.kt [hunk 1]
```diff
@@ -22,11 +25,31 @@ class FragmentMoviesList : Fragment(R.layout.fragment_movies_list) {
...
+
+        view.findViewById<ViewGroup>(R.id.rv_movie)
                 .setOnClickListener {
                     listener?.onItemClickShowDetail()
                 }
+
+    }
+
+    override fun onItemClick(movie: Movie) {
+        listener!!.onItemClickShowDetail()
```
Comment (Axrorxoja) [inline app/src/main/java/com/bignerdranch/android/androidacademy/FragmentMoviesList.kt]: here will be a NULL safe use like so `listener?. onItemClickShowDetail()`

File: app/src/main/java/com/bignerdranch/android/androidacademy/MovieViewHolder.kt [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import android.widget.RatingBar
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+
+class  MovieViewHolder(
+        view: View,
+        private val listener: OnItemClickListener
+        ): RecyclerView.ViewHolder(view) {
+
+    private val img: ImageView? = view.findViewById(R.id.iv_movie)
```
Comment (Axrorxoja) [inline app/src/main/java/com/bignerdranch/android/androidacademy/MovieViewHolder.kt]: why you need nullable views 
when view.findViewById(R.id.iv_movie) return you not nullable views ?

File: app/src/main/java/com/bignerdranch/android/androidacademy/ActorViewHolder.kt [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+
+class ActorViewHolder(itemView: View): RecyclerView.ViewHolder(itemView) {
+    private val avatar = itemView.findViewById<ImageView>(R.id.iv_avatar)
+    private val name = itemView.findViewById<TextView>(R.id.tv_name)
+
+    fun onBind(actor: Actor){
+        avatar!!.setImageResource(actor.img)
```
Comment (Axrorxoja) [inline app/src/main/java/com/bignerdranch/android/androidacademy/ActorViewHolder.kt]: why you use !!(assertion)
it's redundant

File: app/src/main/java/com/bignerdranch/android/androidacademy/FragmentMoviesList.kt (hunk 1 above)
Comment (TimurIsroilov123) [reply app/src/main/java/com/bignerdranch/android/androidacademy/FragmentMoviesList.kt]: got it

File: app/src/main/java/com/bignerdranch/android/androidacademy/MovieViewHolder.kt (hunk 2 above)
Comment (TimurIsroilov123) [reply app/src/main/java/com/bignerdranch/android/androidacademy/MovieViewHolder.kt]: by mistake

File: app/src/main/java/com/bignerdranch/android/androidacademy/ActorViewHolder.kt (hunk 3 above)
Comment (TimurIsroilov123) [reply app/src/main/java/com/bignerdranch/android/androidacademy/ActorViewHolder.kt]: also by mistake. I'll take this kind of mistakes into consideration next time

**Pull request closed without merging**